config/locales/appeal.yml

```diff
@@ -159,31 +77,6 @@ en:
         edit:
           heading: "challenged decision header"
           lead_text: "lead text here change me"
-  activemodel:
```

Comment:
These should be changed to appeal instead of dropped – they're the error messages for the forms.